doc/config.rst

```diff
@@ -14,14 +14,27 @@ With the ``--config`` option you can specify the path to the
 
 The example below will add two rules to the default list of PSR12 set rules:
 
+Consider your codebase as follow:
+
+```
+├── .php-cs-fixer.php
+└── src
+    ├── foo.php
+    ├── bar.php
+    ├── somedir
+    └── maindir
+        ├── subdir
+        └── baz.php
+```
+
```

Comment:
this doesn't work and tbh, I do not see big value of adding it
<img width="483" alt="Screenshot 2023-10-29 at 20 51 52" src="https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/assets/2716794/c1fac8bf-606d-45ca-aad4-28b5638e7cb3">

```suggestion
```